rpmbuild/copr-rpmbuild.spec

```diff
@@ -139,7 +139,15 @@ Requires: yum-utils
 %latest_requires ca-certificates
 %latest_requires distribution-gpg-keys
 %if 0%{?fedora}
+
+%if 0%{?fedora} >= 38
+%latest_requires dnf5
+%latest_requires dnf5-plugins
+%else
 %latest_requires dnf
+%endif
+
+%latest_requires python3-dnf
```

Comment:
I meant `dnf => python3-dnf`.  The thing is that `dnf` is overloaded (on F39 it is provided DNF5, on F38 it is provided by DNF4).  I don't think the current variant is a problem, but dropping the `%latest_reqiures dnf` seems cleaner.

Reply:
> I meant dnf => python3-dnf

I know, but the way I changed it, we will have up-to-date `dnf` on F37, which is what we do now, and since F38 there will be only up-to-date `python3-copr` and the new DNF5 packages.

That IMHO sounds better than `dnf => python3-dnf` because that would unnecessarily remove `dnf` from the set of up-to-date packages that we use on our current builders.

Reply:
Well `/bin/dnf-3` is used by Mock 4.1+ anyway - and updating either `dnf5` or `python3-dnf` or both would trigger the automatic update of `dnf` (whatever provides that).